src/sidebar/components/menu-keyboard-navigation.js

```diff
@@ -20,17 +22,22 @@ export default function MenuKeyboardNavigation({
   visible,
 }) {
   const menuRef = useRef(null);
+  const focusTimer = useRef(null);
```

Comment:
You don't need to use a ref here since you're only referencing the value in the `useEffect` callback. You can just declare a `let focusTimer` variable at the top of that function.

Reply:
I was trying to avoid the `exhaustive-deps` warning about that, but I'll change it, then add that comment.